recipes/cgns/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class CgnsConan(ConanFile):
+    name = "cgns"
+    description = "Standard for data associated with the numerical solution " \
+                  "of fluid dynamics equations."
+    topics = ("conan", "cgns", "data", "cfd", "fluids")
+    homepage = "http://cgns.org/"
+    license = "zlib/libpng"
```

Comment:
we stick the the SPDX short identifiers https://spdx.org/licenses/Zlib.html

```suggestion
    license = "Zlib"
```